src/Sylius/Bundle/ThemeBundle/SyliusThemeBundle.php

```diff
@@ -11,26 +11,60 @@
 
 namespace Sylius\Bundle\ThemeBundle;
 
-use Sylius\Bundle\ThemeBundle\DependencyInjection\Compiler\ThemeRepositoryPass;
-use Sylius\Bundle\ThemeBundle\Translation\DependencyInjection\Compiler\ThemeAwareLoaderDecoratorPass;
+use Sylius\Bundle\ResourceBundle\AbstractResourceBundle;
+use Sylius\Bundle\ResourceBundle\SyliusResourceBundle;
+use Sylius\Bundle\ThemeBundle\Model\ThemeAuthor;
+use Sylius\Bundle\ThemeBundle\Model\ThemeAuthorInterface;
+use Sylius\Bundle\ThemeBundle\Model\ThemeInterface;
+use Sylius\Bundle\ThemeBundle\Translation\DependencyInjection\Compiler\TranslatorAliasingPass;
+use Sylius\Bundle\ThemeBundle\Translation\DependencyInjection\Compiler\TranslatorLoaderProviderPass;
 use Sylius\Bundle\ThemeBundle\Translation\DependencyInjection\Compiler\ThemeAwareSourcesPass;
+use Sylius\Bundle\ThemeBundle\Translation\DependencyInjection\Compiler\TranslatorResourceProviderPass;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\HttpKernel\Bundle\Bundle;
 
 /**
  * @author Kamil Kokot <kamil.kokot@lakion.com>
  */
-class SyliusThemeBundle extends Bundle
+class SyliusThemeBundle extends AbstractResourceBundle
 {
     /**
+     * {@inheritdoc}
+     */
+    public static function getSupportedDrivers()
+    {
+        return [
+            SyliusResourceBundle::DRIVER_DOCTRINE_ORM,
+        ];
+    }
+
+    /**
      * @param ContainerBuilder $container
      */
     public function build(ContainerBuilder $container)
     {
         parent::build($container);
 
-        $container->addCompilerPass(new ThemeRepositoryPass());
-        $container->addCompilerPass(new ThemeAwareSourcesPass());
-        $container->addCompilerPass(new ThemeAwareLoaderDecoratorPass());
+        $container->addCompilerPass(new TranslatorAliasingPass());
+        $container->addCompilerPass(new TranslatorLoaderProviderPass());
+        $container->addCompilerPass(new TranslatorResourceProviderPass());
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getModelInterfaces()
+    {
+        return [
+            ThemeInterface::class => 'sylius.model.theme.class',
+            ThemeAuthorInterface::class => 'sylius.model.theme_author.class',
+        ];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getModelNamespace()
+    {
+        return 'Sylius\Bundle\ThemeBundle\Model';
```

Comment:
@Zales0123 It's a namespace, not a class :)

@steffenbrem it is, but it would be really confusing to call `::class` on a namespace (not existing class really), PHP won't throw any error as it does not check if the class exists, but every IDE would highlight it :)